software-modules/internet/chromium/chromium.sh

```diff
@@ -20,12 +20,18 @@ TARGET_DIR="/run/initramfs/memory/system/huronOS/software/internet/"
 
 ## Install software
 apt update
-apt install --yes --no-install-recommends $NAME
+# TODO: When migrating to debian12, replace pip with python3-watchdog and remove the following two lines
+apt install --yes --no-install-recommends $NAME python-is-python3 pip
+pip install watchdog
```

Comment:
If  both chromium and firefox needs watchdog why not to include it on the shared-libs instead?

Reply:
You're right, I forgot to move that, already moving